### DISCUSSION TITLE
Emgu.CV.runtime.windows.dldt, x64 and x86 folders have no dlls

### DISCUSSION THREAD

**Discussion by erecord** [emgucv/emgucv#427] [Q&A]
Hi,

I get an error message when trying to use Emgu v4.5.1.4349. The error message says it failed to find cvextern.dll

I have installed the following packages from NuGet

- Emgu.CV v4.5.1.4349
- Emgu.CV.Bitmap v4.5.1.4349

I have installed the following packages from MyGet (commercial repository)

- Emgu.CV.runtime.windows.dldt v4.5.1.4349

---

However, when I look at the bin directories, the x64 and x86 folders can be found but they are empty inside. 

Whereas v4.4.04099 these folders have all the dll inside them. This seems to have broken when the x64 and x86 were split into two extra packages in v4.5.

Can you help me resolve the error message I get when trying to use EmguCV v4.5.1.4349?

Thanks

**Comment by emgucv** [comment]
In the 4.5.1 release, for the nuget packages, the x64 runtime files has been relocated from "x64" folder into "runtimes\win-x64\native". The x86 runtime files has been relocated from "x86" fodler into "runtimes\win-x86\native". 

This is done to make sure Emgu CV complys with nuget requirements:
https://docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks
For details, browse down to the "Architecture-specific folders" sections of the above page.

If you run into issue loading cvextern.dll, can you post the full error log?

To check missing dependency, if you are running in 64bit mode, please use the 64bit dependency walker to open the "cvextern.dll" file under the "runtimes\win-x64\native" subfolder, from the folder of the executable.
